MastodonToTwitter.py

```diff
@@ -39,7 +38,7 @@
 TWITTER_RETRY_DELAY = 20
 
 # Media regex, to trim media URLs
-MEDIA_REGEXP = re.compile(re.escape(MASTODON_BASE_URL.rstrip("/")) + "\/media\/(\w)+(\s|$)+")
+MASTODON_BASE_URL = ''
```

Comment:
I wasn't sure at first, so I left it in. Checking it now, it looks like it's declared separately on creating a new app, or loading from secrets. Removed.